### PR TITLE
Upgrade to Rails 5.0.1

### PR DESCRIPTION
**Pull request by dwhenry**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
-# Pick the frameworks you want:
-require 'active_model/railtie'
-require 'active_job/railtie'
-require 'active_record/railtie'
-require 'action_controller/railtie'
-require 'action_mailer/railtie'
-require 'action_view/railtie'
-require 'sprockets/railtie'
-# require "rails/test_unit/railtie"
+require 'rails/all'
```
Comment (benlovell) [inline config/application.rb]: Do we want to load all of Rails here? I assume this loads Action Cable too now.

File: config/environments/production.rb [hunk 2]
```diff
@@ -14,15 +14,9 @@
...
-  # Enable Rack::Cache to put a simple HTTP cache in front of your application
-  # Add `rack-cache` to your Gemfile before enabling this.
-  # For large-scale production use, consider using a caching reverse proxy like
-  # NGINX, varnish or squid.
-  # config.action_dispatch.rack_cache = true
-
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
```
Comment (benlovell) [inline config/environments/production.rb]: Can we add the headers for caching here too?

**Comment by dwhenry** [comment]
@benlovell changes made. :)

**Comment by benlovell** [comment]
Well played 💯

**Pull request merged**